src/lib.rs

```diff
@@ -106,6 +108,8 @@ valkey_module! {
             ["bloom-memory-limit-per-filter", &*configs::BLOOM_MEMORY_LIMIT_PER_FILTER, configs::BLOOM_MEMORY_LIMIT_PER_FILTER_DEFAULT, configs::BLOOM_MEMORY_LIMIT_PER_FILTER_MIN, configs::BLOOM_MEMORY_LIMIT_PER_FILTER_MAX, ConfigurationFlags::DEFAULT, None],
         ],
         string: [
+            ["bloom-fp-rate", &*configs::BLOOM_FP_RATE, &*configs::BLOOM_FP_RATE_DEFAULT_STRING, ConfigurationFlags::DEFAULT, Some(Box::new(configs::on_string_config_set))],
+            ["bloom-tightening-ratio", &*configs::TIGHTENING_RATIO_CONFIG, &*configs::TIGHTENING_RATIO_DEFAULT_STRING, ConfigurationFlags::DEFAULT, Some(Box::new(configs::on_string_config_set))],
```

Comment:
This is not implementing a config set handler. It is implementing a config on change handler. This is how you implement a config set handler:
```suggestion
            ["bloom-tightening-ratio", &*configs::TIGHTENING_RATIO_CONFIG, &*configs::TIGHTENING_RATIO_DEFAULT_STRING, ConfigurationFlags::DEFAULT, None, Some(Box::new(configs::on_string_config_set))],
```